classes/statistics/PKPMetricsDAO.inc.php

```diff
@@ -276,7 +276,7 @@ function insertRecord($record) {
 		$assocId = $recordToStore['assoc_id'] = (int)$recordToStore['assoc_id'];
 
 		list($contextId, $pkpSectionId, $assocObjType,
-			$assocObjId, $submissionId, $representationId) = $this->foreignKeyLookup($assocType, $assocId);
+			$assocObjId, $submissionId, $representationId) = $this->foreignKeyLookup($assocType, $assocId, $record['representation_id']);
```

Comment:
Does this need to be added to the `$requiredDimensions` check just above this line? Or if not, do you need to check `isset()` first?

Reply:
Oh, yes, I should check if isset()... :+1: